### ISSUE TITLE
Random 3-20 minute disconnects from server

### ISSUE THREAD

**Issue by Fryman3000** [dalaranwow/dalaran-wow#5289]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
On Dalaran-WoW I am being randomly disconnected every 3-20 minutes with no rhyme or reason, I am currently on Wi-Fi, Algalon server. I don't believe it is a server disconnect as other friends have been playing just fine, actions I engage it when disconnects happen so far: Open chat, using professions (tailoring/Enchanting), minimally idling, and movement
**Expected behaviour**: Tell us what should happen instead
Game should NOT unexpectedly disconnect me from server 
**Steps to reproduce the problem**:

1. Login to my account
2. Select my character
3.  Normal gameplay

**Include proofs for this behaviour**
Ran a tracert

Tracing route to dalaran-wow.com [104.24.1.114]
over a maximum of 30 hops:

  1     *        *        *     Request timed out.
  2     *        *        *     Request timed out.
  3     *        *        *     Request timed out.
  4     *        *        *     Request timed out.
  5     *        *        *     Request timed out.
  6     *        *        *     Request timed out.
  7     *        *        *     Request timed out.
  8     *        *        *     Request timed out.
  9    12 ms    12 ms    10 ms  104.24.1.114 

Trace complete.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by punims** [comment]
github is meant for ingame coding issues I believe, not for server issues... not much a programmer can do about it.

**Comment by Chenocide** [comment]
There's a lot of people with really random connectivity problems when it comes to Dalaran-wow, not sure why though.
I believe server connections and routing problems would fit just fine but one of the collaborators could clarify that.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report - it is not server's issue, else everyone will experience it and there are thousands of players without issues every day. The most common problem regarding packet routing is a malfunction on a network node from which your traffic is passing through to reach Algalon's host. Of course there could be many other problems either on your local network or Data Center, but the mentioned is the usual. I doubt is a Data Center issue, because a bigger subset of players from your country / area would experience the same than you.

If this is the case, you can do two things: wait for the network to get a safe path from which route your traffic through, or use a VPN in the meanwhile from another location (E.g: London, Paris).

There could be a much simpler answer to your problem but I guess you have checked them all (corrupt client, P2P/Torrent or other software that may affect certain ports, modified addons...etc). I have seen players with such issues during the last years that thought it was the server, but they ended fixing by themselves after a thorough research on their end (random disconnects, MS increase on specific situations etc.).

Anyways, I hope you achieve to fix the issue - if you do, feedback would be welcome so players in the future can have a reference.

Best wishes.
